ISISScientificComputing/autoreduce
# Add accessibility statement AR-1200

### Summary of work
- [x] Create accessibility statement page
- [x] Write page selenium tests
- [x] As a team agree on the contents of the page
- [x] Get this checked over by UKRI's legal team
  - [x] Sent
  - [x] Get confirmation

### How to test your work
- Tests pass
- Code review
- Grammar check
- Read over the contents of the page and agree or disagree about the contents of the page. Particularly review the sections: 
  - How accessible this website is
  - Non-accessible content
  - Disproportionate burden
- Review with the below pages. Agree or disagree that striked though sections should be removed with reference to:
  - https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website
  - https://www.gov.uk/guidance/make-your-website-or-app-accessible-and-publish-an-accessibility-statement
- Agree on plan to fix accessibility issues in the section "Non-compliance with the accessibility regulations"

### Additional comments
* Page is based on this statement here https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website and also https://www.isis.stfc.ac.uk/pages/accessibility.aspx


Jira: https://autoreduce.atlassian.net/browse/AR-1200?atlOrigin=eyJpIjoiN2MzY2I3MjRiZjc3NDQ2ODljYmRjYzI5MTQyNzQ2NDgiLCJwIjoiaiJ9

Fixes #1200


**Before merging ensure the release notes have been updated**
